resources/views/admin/clinic_types/create.blade.php

```diff
@@ -0,0 +1,23 @@
+@extends('admin.layouts.app')
+@section('title')
+  Add Clinic Type
+@endsection
+@section('content')
+<!-- /.col-lg-12 -->
+        <div class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pt-3 pb-2 mb-3 border-bottom">
+            <h1 class="h2">Add Clinic Type</h1>
+          </div>
+        @include('admin.layouts.partials.block-error')
+        <form action="{!! route('admin.clinic-types.store') !!}" method="POST" class="col-md-6">
```

Comment:
Dùng `{{ }}` nha
https://stackoverflow.com/questions/35030977/what-is-the-difference-between-and-in-laravel-blade-files